### ISSUE TITLE
[BUG] VM bytecode logical error

### ISSUE THREAD

**Issue by DotRacel** [levno-710/Prometheus#18]
**Describe the bug**
as the title says

**Expected behavior**
returns a wrong value

**To Reproduce**
```lua
local a, b = "aaa", "aaa"
local c = a == b == true
print(c)
```
using vmify

**Additional context**
it is supposed to print "true", but it returns "false"

**Comment by levno-710** [comment]
I will look into it

**Comment by levno-710** [comment]
# Fixed
I managed to fix the bug in the Parser.
The fix is included in release v0.1.8.
